spikes/lk_led_/main.cpp

```diff
@@ -0,0 +1,76 @@
+// Leka - LekaOS
+// Copyright 2022 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#include <cstddef>
+
+#include "PinNames.h"
+
+// #include "drivers/BufferedSerial.h"
+// #include "drivers/HighResClock.h"
+// #include "rtos/Kernel.h"
+#include "rtos/ThisThread.h"
+
+#include "CoreLED.h"
+#include "CoreSPI.h"
+#include "HelloWorld.h"
+#include "LogKit.h"
+
+using namespace leka;
+using namespace std::chrono;
+
+auto const NUM_BELT_LEDS		= 20;
+auto const NUM_EAR_LEDS			= 2;
+auto const BRIGHTNESS			= 255;
+const std::array<RGB, 6> COLORS = {
+	RGB::pure_green, RGB::pure_red, RGB::pure_blue, RGB::yellow, RGB::cyan, RGB::magenta,
+};
+
+void ChangeColor(CoreLED coreled, uint8_t index)
```

Comment:
c'est mieux de passer les objects comme ça par référence, ça évite une copie.

```suggestion
void changeColor(CoreLED& coreled, uint8_t index)
```